dask/dataframe/tseries/tests/test_resample.py

```diff
@@ -213,3 +214,8 @@ def test_common_aggs(agg):
     expected = f(ds.resample("1d"))
 
     assert_eq(res, expected, check_dtype=False)
+
+
+def test_getnanos_deprecated():
+    with pytest.warns(FutureWarning, match="getnanos is deprecated"):
```

Comment:
The warning message is slightly different after adding `after_version=` to `_deprecated`

```suggestion
    with pytest.warns(FutureWarning, match="getnanos was deprecated"):
```